packages/duckdb/test_duckdb.py

```diff
@@ -0,0 +1,17 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide(packages=["duckdb", "pandas"])
+def test_duckdb(selenium):
+    import duckdb
+    import pandas as pd
+
+    example_df = pd.DataFrame({"col_1": [1, 2, 3, 4, 5], "col_2": [10, 20, 30, 40, 50]})
```

Comment:
This example_df is never being used according to flake8, due to the string usage (wow, that looks odd). Needs a noqa added.

Reply:
There's another approach that keeps the variable used and looks cleaner where the table is explicitly registered so I've changed over to that.

Reply:
That looks much better!